iib/workers/tasks/build.py

```diff
@@ -340,21 +350,22 @@ def _get_present_bundles(from_index, base_dir):
         return [], []
 
     # Transform returned data to parsable json
-    unique_present_bundles = []
+    unique_present_bundles: List[BundleImage] = []
     unique_present_bundles_pull_spec = []
     present_bundles = get_bundle_json(bundles)
-
```

Comment:
This was probably removed by accident? Could you please add the empty line back?